templates/Start/vouchers.html.twig

```diff
@@ -42,5 +42,10 @@
                 {% endif %}
             {%  endif %}
         </div>
+        <div class="col-xs-12 col-sm-6 col-md-6">
```

Comment:
That should be under the headline, so you should use a full width row for headline and a new row with two columns for the content and the description.